homu/html/queue.html

```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="utf-8">
         <title>Homu queue - {{repo_name}}</title>
+        <link href="//cdn.datatables.net/1.10.4/css/jquery.dataTables.min.css">
```

Comment:
On a lark, I just completely removed it, and everything seems to be working fine?